source/server/admin/admin.cc

```diff
@@ -130,6 +132,11 @@ absl::optional<std::string> maskParam(const Http::Utility::QueryParams& params)
   return Utility::queryParam(params, "mask");
 }
 
+// Helper method to get the eds parameter.
+bool edsParam(const Http::Utility::QueryParams& params) {
+  return Utility::queryParam(params, "includeEds") != absl::nullopt;
```

Comment:
This should probably be documented in https://www.envoyproxy.io/docs/envoy/latest/operations/admin#get--config_dump.

Reply:
Updated the docs [here](https://github.com/envoyproxy/envoy/pull/11577/files#diff-338233d0fae252940b3aae4de8c1135d)